### PR TITLE
Reword maintenance policy sentences

### PR DESCRIPTION
**Pull request by 4383**
Reword sentences to make them more meaningful, fix typos, remove examples to discourage new usages.

### CODE REVIEW

**Comment by 4383** [comment]
@os-frickler: o/ here is the following patches

File: README.rst [hunk 1]
```diff
@@ -13,18 +13,18 @@ implementation.
 This gap is now too high and can lead you to unexpected side effects and bugs
 in your applications.
 
-**Eventlet is now switching to legacy mode**. **Only maintenance for stability
-and bug fixing** will be provided. **No new features will be accepted**, except
-those related to the asyncio migration. **Usage in new projects are
-discouraged**. **Our goal is to plan the retirement of eventlet** and to give
-you ways to move away from eventlet.
+Eventlet now follow a new maintenance policy. **Only maintenance for
```
Comment (osfrickler) [inline README.rst:16]: follows

**Review by osfrickler — COMMENTED** [review]
Thx, apply the same changes to doc/index.rst?

File: README.rst [hunk 2]
```diff
@@ -13,18 +13,18 @@ implementation.
...
 in your applications.
 
-**Eventlet is now switching to legacy mode**. **Only maintenance for stability
-and bug fixing** will be provided. **No new features will be accepted**, except
-those related to the asyncio migration. **Usage in new projects are
-discouraged**. **Our goal is to plan the retirement of eventlet** and to give
-you ways to move away from eventlet.
+Eventlet now follows a new maintenance policy. **Only maintenance for
+stability and bug fixing** will be provided. **No new features will be
+accepted**, except those related to the asyncio migration. **Usages in new
```
Comment (itamarst) [inline README.rst:18]: Maybe "Using eventlet in new projects is discouraged."?

**Review by itamarst — APPROVED** [review]

File: README.rst (hunk 2 above)
Comment (4383) [reply README.rst:18]: let's update this sentence later if really needed.

**Pull request merged**